packages/platform/atoms/booker/BookerPlatformWrapper.tsx

```diff
@@ -73,11 +75,24 @@ export type BookerPlatformWrapperAtomProps = Omit<BookerProps, "username" | "ent
   onDeleteSlotSuccess?: (data: ApiSuccessResponseWithoutData) => void;
   onDeleteSlotError?: (data: ApiErrorResponse) => void;
   locationUrl?: string;
-  teamId?: number;
 };
 
-export const BookerPlatformWrapper = (props: BookerPlatformWrapperAtomProps) => {
+type BookerPlatformWrapperAtomPropsForIndividual = BookerPlatformWrapperAtomProps & {
+  username: string | string[];
+  isTeamEvent?: false;
+};
+
+type BookerPlatformWrapperAtomPropsForTeam = BookerPlatformWrapperAtomProps & {
+  username?: string | string[];
+  isTeamEvent: true;
```

Comment:
Idea for future - I think we can simplify by inferring that if `teamId` is passed to set `isTeamEvent` to true.

Reply:
good idea